### PR TITLE
Document how to debug QE pipeline failures

### PR DESCRIPTION
**Pull request by cipherboy**
`Signed-off-by: Alexander Scheel <ascheel@redhat.com>`

### CODE REVIEW

File: docs/development/Debugging_Pipeline.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+here, switch to the `runner` user:
+
+    # sudo su runner
+    $ cd
+
+In `$HOME/work/pki/pki/` is the cloned GitHub repo, plus Vagrant home. There
+are two VMs currently provisioned in this pipeline:
+
+    - `controller` at IP address `192.168.33.10`, which runs Ansible
+    - `master` at IP address `192.168.33.20`, which runs PKI (and is
```
Comment (SilleBille) [inline docs/development/Debugging_Pipeline.md:19]: Markdown considers this as a code block. Can you remove the spaces before the `-`?

File: docs/development/Debugging_Pipeline.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Debugging QE Pipeline Failures
+
+When QE pipeline fails, the console output will display a ssh address
+such as:
```
Comment (SilleBille) [inline docs/development/Debugging_Pipeline.md:4]: Can you add the job name to this line? `Setup tmate session`

**Review by SilleBille — COMMENTED** [review]
(pokes my nose into this PR)
You may also want to tell that the `Setup tmate session` job times out after ~6 hours approx.

File: docs/development/Debugging_Pipeline.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+    # sudo su runner
+    $ cd
+
+In `$HOME/work/pki/pki/` is the cloned GitHub repo, plus Vagrant home. There
+are two VMs currently provisioned in this pipeline:
+
+ - `controller` at IP address `192.168.33.10`, which runs Ansible
+ - `master` at IP address `192.168.33.20`, which runs PKI (and is
+   the target of Anisble playbooks.
```
Comment (SilleBille) [inline docs/development/Debugging_Pipeline.md:20]: (nitpick) Brackets and Spelling error for Ansible 😝

**Review by SilleBille — APPROVED** [review]

**Comment by cipherboy** [comment]
Thanks @SilleBille!

**Pull request merged**